README.md

```diff
@@ -0,0 +1,38 @@
+# awx
+
+This role sets up a local AWX installation using the Docker installation.
+Non-containerized installation is not supported by AWX as of writing.
+
+The data can be found in `/var/lib/awx`.
+
+## Requirements
+
+Ubuntu
+
+## Role Variables
+
+| Name                | Default/Required    | Description                                      |
+|---------------------|:-------------------:|--------------------------------------------------|
+| `awx_arch`          | `amd64`             | Architecture of the target CPU                   |
+| `awx_postgres_data` | `/var/lib/postgres` | Location of the data of the PostgreSQL container |
```

Comment:
This var is not used. Did you intend to add our postgres role as a dependency and somehow use it here?

Reply:
No, AWX comes with a postgresql container

Reply:
Still, I can't see where the var is used?